### DISCUSSION TITLE
Added IDEAS.md to show future possible additions

### DISCUSSION THREAD

**Discussion by DustinBrett** [DustinBrett/daedalOS#463] [Announcements]
I've added a new file which gave me an excuse to dump my 4 years of bookmarks into something more organized. Most of the stuff in there are things I'd like to eventually get into daedalOS. I figured I'd mention it here so people can see what other ideas I have for the future of this project.

https://github.com/DustinBrett/daedalOS/blob/main/IDEAS.md

**Comment by Ser-Gen** [comment]
Thank you so much, your work inspires me!

That's it with this file too, I also ended up posting my reference lists similarly to you, in a separate repository.
There are a lot of links in there, I'm thinking about how to make it more convenient. I was planning to add more structure, but this is it for now.
A small amount of titles and descriptions in Russian, if it will cause problems, I can process it too, let me know.

I hope that at least some of my lists will be as useful to you as yours are to me.

https://github.com/Ser-Gen/bookmarks

**Reply by DustinBrett** [reply]
Thanks for sharing, I will check it out!
